code/game/objects/items/ego_weapons/he.dm

```diff
@@ -516,3 +516,110 @@
 	icon_state = "pleasure"
 	happy = FALSE
 	force = 30
+
+/obj/item/ego_weapon/giant
```

Comment:
Normally we are against EGO higher than their abnormalities, but this one is okay because it's mostly a defensive one. Reduce force to 22 to keep it the same as a teth.

Reply:
Look at the attack speed